apps/system/utils/netcmd.c

```diff
@@ -95,6 +95,95 @@ struct ifconfig_cmd_info_s {
 extern int netdb_main(int argc, char *argv[]);
 #endif
 
+int getIp(void)
```

Comment:
I think we can separate this change into another commit, because, this is not for CSI example.
It can be much better to manage the code.

Reply:
Hi, this is needed for CSI example, as it require both wifi connected and IP detected to run. Thanks